docs/ref/contrib/admin/index.txt

```diff
@@ -1853,6 +1853,23 @@ templates used by the :class:`ModelAdmin` views:
         def get_changeform_initial_data(self, request):
             return {'name': 'custom_initial_value'}
 
+.. method:: ModelAdmin.get_deleted_objects(objs, opts, request, using)
+
+    A hook for getting all objects related to ``objs`` that should also be
+    deleted on the admin delete view and in the default ``delete_selected``
+    action.
+
+    Return a nested list of strings suitable for display in the
+    template with the ``unordered_list`` filter.
+
+    By default this returns ``get_deleted_objects()`` from ``django.contrib.admin.utils``.
+
+    ``objs`` is a homogeneous iterable of objects (e.g. a QuerySet).
+
+    .. versionchanged:: 2.0
```

Comment:
Could you please use [`versionadded`](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded) instead of `versionchanged`.